From f07c73b3dfdc7dc087d681b8687c19621d640317 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 8 Oct 2014 14:05:11 -0400
Subject: [redhat] kabi: Adding symbol root_device_unregister (drivers/base/core.c)

Message-id: <d112a96b47de30fdd4511cd27b7aa792e76de45f.1412771859.git.poros@redhat.com>
Patchwork-id: 97201
O-Subject: [RHEL7.1 PATCH 51/67] kabi: Adding symbol root_device_unregister (drivers/base/core.c)
Bugzilla: 1148035
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

Bugzilla: 1148035
    https://bugzilla.redhat.com/show_bug.cgi?id=1148035

    Description:
    ============
    Used to clean up if an error occures durring root_device_unregister
    and when shuting down sysfw interface.
    Symbol used in hwperf.ko

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/root_device_unregister
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/root_device_unregister
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/root_device_unregister

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/root_device_unregister b/redhat/kabi/kabi-module/kabi_ppc64/root_device_unregister
new file mode 100644
index 0000000..e3e1c08
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/root_device_unregister
@@ -0,0 +1,2 @@
+#1-
+0xedc3b416 root_device_unregister vmlinux EXPORT_SYMBOL_GPL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/root_device_unregister b/redhat/kabi/kabi-module/kabi_s390x/root_device_unregister
new file mode 100644
index 0000000..2885d9e
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/root_device_unregister
@@ -0,0 +1,2 @@
+#1-
+0x017dcb44 root_device_unregister vmlinux EXPORT_SYMBOL_GPL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/root_device_unregister b/redhat/kabi/kabi-module/kabi_x86_64/root_device_unregister
new file mode 100644
index 0000000..c5999d4
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/root_device_unregister
@@ -0,0 +1,2 @@
+#1-
+0x1a867ca0 root_device_unregister vmlinux EXPORT_SYMBOL_GPL
-- 
1.7.1